mrkingz/Maintenance-Tracker
# Bg fix dropdown bug 157554128

#### What does this PR do?
Fix bug in dropdown menu

#### Description of Task to be completed?
Align the dropdown content

#### How should this be manually tested?
Clone the ui folder in the repo, launch the index.html, navigate to  home.html by the clicking sign in button; finally, click the menu button, the dropdown should be properly align

#### Any background context you wan to provide?
N/A

#### What are the relevant Pivotal Tracker stories?
- 157554128

#### Screenshots (if appropriate)
N/A
